blitz.py

```diff
@@ -274,7 +281,7 @@ def game_stage():
             current_stage = 20
         else:
             current_stage = 30
-    else:
+    elif current_stage == 0:
```

Comment:
I don't know what this code is doing well enough to tell if this could cause a problem or not. Do you mind adding a comment about why this change was made?

Reply:
This is the main reason of the PR. If you follow the logic below, you can see that some of the branches above set `current_stage` but not `stages`. The else iterates over `stages`, which causes an exception to be raised (captured by a try/except higher up in the stack). My assumption was that the cases when it explicitly sets `current_stage`, it shouldn't be overwritten.

I don't know if it could potentially cause issues, but the code still appears to work (it plays blitz) and it no longer throws 900 exceptions a second clogging up the logs.

Reply:
+1 makes sense, thanks.